### PR TITLE
fix: styling and layout changes for dashboard, footer and product page

### PR DESCRIPTION
**Pull request by arslanashraf7**
#### Pre-Flight checklist

- [x] Screenshots and design review for any changes that affect layout or styling
  - [x] Desktop screenshots
  - [x] Mobile width screenshots
- [x] Testing
  - [x] Code is tested
  - [x] Changes have been manually tested

#### What are the relevant tickets?
#91 

#### What's this PR do?
- Fixes the styling/layout issues in the footer
- Fixes the layout issues on the dashboard
- Fixes the layout issues on the Product Detail Page
- Removed extra <p> tags from rich text in product detail page
- Removed xPRO's unused style from the footer which was copied over xPRO

Putting simply, these are the things from PDF:
1. All footer text to be `font-weight:400` & spacing Mentioned in PDF
2. For the dashboard page, adjusted the spacing mentioned in the PDF
3. For product page background is set to `white` and spacing adjusted as per PDF

**Note:** There is a pdf file attached to the associated issue which will help out identifying the required styling changes

#### How should this be manually tested?
- Verify it against the changes mentioned in the PDF attached to the ticket.
- Verify the nothing else has been affected by this PR

#### Where should the reviewer start?
- HomePage

#### Screenshots (if appropriate)
**Footer Desktop**
Screenshot 2021-08-17 at 5 00 17 PM

**Footer Mobile**
Screenshot 2021-08-17 at 5 00 30 PM

**Dashboard Desktop**
Screenshot 2021-08-17 at 5 00 57 PM

**Dashboard Mobile**
Screenshot 2021-08-17 at 5 01 07 PM

**Product Page Desktop**
Screenshot 2021-08-17 at 5 28 24 PM

**Product Page Mobile**
Screenshot 2021-08-17 at 5 30 30 PM

### CODE REVIEW

File: static/scss/dashboard.scss [hunk 1]
```diff
@@ -37,4 +37,18 @@ $featureImgMobileWidth: 343px;
       }
     }
   }
-}
\ No newline at end of file
+}
+
+.courses-heading {
+  font-size: 40px;
```
Comment (gsidebo) [inline static/scss/dashboard.scss:43]: This font-size should be applied to the general style in `common.scss` so all `h1`'s use it. There is no reason to have some `h1` elements at 40px and others at 42px/48px. Those values are close enough that you should always assume that the change should be applied to all elements like it. You can also try applying the mobile styles to the `common.scss` and see how the other `h1` elements on other pages look.

You can also remove the `courses-heading` class from that `h1` if the above suggestions work out

**Review by gsidebo — CHANGES_REQUESTED** [review]
There are a couple other possible refactors, but this is good to merge after the `h1` thing is addressed

**Comment by arslanashraf7** [comment]
>This font-size should be applied to the general style in common.scss so all h1's use it. There is no reason to have some h1 elements at 40px and others at 42px/48px. Those values are close enough that you should always assume that the change should be applied to all elements like it. You can also try applying the mobile styles to the common.scss and see how the other h1 elements on other pages look.

I like this idea and I've updated the font size/ lint-height styles `common.scss` for `h1`. Instead of setting `40px` for all `h1` i took the liberty of setting `font-size/line-height` as `42px/52px` for desktops and `34px/38px` for mobile. The reason I tried to set `42px` instead of `40px` was to make sure that we could make the h1 style common keeping it close to the existing design and also being able to do the required styling that's part of this ticket.

Attaching some before/after screenshots below so that you can also take a look at the difference:

Screenshot 2021-08-23 at 3 54 50 PM
Screenshot 2021-08-23 at 3 54 58 PM

Screenshot 2021-08-23 at 3 55 11 PM
Screenshot 2021-08-23 at 3 55 20 PM

Screenshot 2021-08-23 at 3 52 34 PM
Screenshot 2021-08-23 at 3 52 26 PM

Screenshot 2021-08-23 at 3 52 52 PM
Screenshot 2021-08-23 at 3 52 57 PM

Screenshot 2021-08-23 at 3 53 18 PM
Screenshot 2021-08-23 at 3 53 26 PM

Screenshot 2021-08-23 at 3 53 45 PM
Screenshot 2021-08-23 at 3 53 40 PM
Screenshot 2021-08-23 at 4 30 19 PM
Screenshot 2021-08-23 at 4 30 11 PM


>You can also remove the courses-heading class from that h1 if the above suggestions work out

- I've removed the size and line height-related styling from the class but I've kept the class for spacing-related styles that in my opinion might change in different places e.g. specifically on the dashboard.

If you are fine with these changes we can merge them or if you think the design changes are too broad I will revert the last commit.


CC: @pdpinch (Tagging because this might introduce some design tweaks, not very major but wanted to let you know in any case)

**Comment by pdpinch** [comment]
I'm checking with the designer why the course detail page has a larger H1 than the other account creation pages.

**Comment by pdpinch** [comment]
InVision has been updated, https://projects.invisionapp.com/d/main?origin=v7#/projects/prototypes/21535972?inbox=true&inboxProjectID=21535972&inboxConversationID=144914028

In any case, the h1 on the course detail should be the same size as all the rest.

**Review by gsidebo — APPROVED** [review]
Looks good 👍 . You can still get rid of the `courses-heading` class as I suggested before, and just style the `h1` directly in `dashboard.scss` to set the margins. Feel free to merge after you do that

**Pull request merged**